scss/_reboot.scss

```diff
@@ -727,3 +725,26 @@ progress {
 [hidden] {
   display: none !important;
 }
+
+// Boosted mod
+[class*="-dark"],
+[class*="bg-secondary"] {
+  --link-color: #{$link-color-dark};
+  --link-hover-color: #{$link-hover-color-dark};
+  --caption-color: #{$table-caption-color-dark};
+  --code-color: #{$code-color-dark};
+  --kbd-color: #{$kbd-color-dark};
+  --kbd-bg: #{$kbd-bg-dark};
+  --pre-color: #{$pre-color-dark};
+
+  [class*="bg-"]:not(&) {
```

Comment:
Could also be `[class*="bg-"]:not(&):not(.bg-transparent)` instead, in order to manage some edge cases.